lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_completecodefreeze_prechecks.rb

```diff
@@ -0,0 +1,67 @@
+module Fastlane
+    module Actions
+      class AndroidCompletecodefreezePrechecksAction < Action
+        def self.run(params)
+          UI.message "Skip confirm: #{params[:skip_confirm]}"
+          
+          require_relative '../../helper/android/android_version_helper.rb'
+          require_relative '../../helper/android/android_git_helper.rb'
+  
+          UI.user_error!("This is not a release branch. Abort.") unless other_action.git_branch.start_with?("release/")
```

Comment:
Shouldn't we even go as far as checking that the branch is actually named `"release/#{version}"` (with `version` variable defined as line 12) – to be sure we're not accidentally on the release branch for another version and that the version provided via code and returned by `get_public_version` matches the target version number?

Reply:
Nice idea.

Reply:
We could... but this will kill any easy way to test future changes 😂. This, and the fact that we haven't had issues with the looser check, are the main reasons why we never added stricter checks.